### PR TITLE
Updates to serve_local.sh and context documentation

### PR DESCRIPTION
**Pull request by Nlar**
serve_local.sh was Python 2 specific and would not run under Python 3.  Updated so that it ran under Python 3.  Removed echo statement as output is provided while running the command.

I could not find documentation that shows how to add a task to a context.  Updated context.html as a result.

### CODE REVIEW

File: html/docs/context.html [hunk 1]
```diff
@@ -192,13 +192,20 @@ <h4>Deleting a Context</h4>
               it, the context is cleared.
             </p>
 
+            <a name="add"></a>
+            <h4>Adding Task to a Context</h4>
+            <p>
+              To add a task to the <code>work</code> context:
+            <pre>$ task add +work Tell co-workers about Taskwarrior</pre>
+            </p>
+
```
Comment (tbabej) [inline html/docs/context.html:201]: I believe this could lead to a confusion for our users, namely, lead people to assume a one-to-one correspondence between tags (`+work`) and the context (`work`).

While this works if context is defined directly as the tag, i.e.:
```
task context define work +work
```

Contexts can be arbitrary filters however, for example:
```
task context define todayswork +work due:today
```

In this case, neither doing `task add +work thing I need to do today` or `task add +todayswork thing I need to do today` would "create the task in the context", to match the filter that the context `todayswork` imposes, one would have to do:

```
task add due:today +work A thing I need to do today
```

In other words, to "create task in the context", one needs to create a task whose metadata will match the filter the context imposes. Sometimes that might be also straight impossible, consider the following context:
```
task context define oldtasks +PENDING entry.before:now-3y
```

which would apply a filter for all pending tasks that were entered into Taskwarrior more than 3 years ago. A new task will not match that filter, since it's entry attribute (the date time of creation) will be more recent.

**Comment by tbabej** [comment]
For more context on "writeable context" see #2402 and #1955.

File: html/docs/context.html (hunk 1 above)
Comment (Nlar) [reply html/docs/context.html:201]: I agree this is confusing now that I understand better.  Thank you for going through that!  The page examples list:
``` $ task context define work +work or +freelance ```
I did not connect that the ```+work ``` was a basic tag and not a the descriptor being called tag as part of the explanation.  Would a link the tags page (tags.html) as part of the context be helpful. It is reasonable that I am the exception and the documentation is clear enough.  

Would adding an example using complex filters such as above be worthwhile?  Possibly mention you shouldn't try to assign to them directly.  I managed to create some oddness my task list (I'll file a bug if it is reproducible) through direct assignment.  

Apologies if I should be moving this to discussion at this point.

**Comment by Nlar** [comment]
Per discussion, closing request.
